CodeMaid/UI/Converters/AppendParametersConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
```

Comment:
I don't think this class is needed at all when the TabControl is taken out of the ViewModel (comments below).